.github/workflows/test-loadgen.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Test for MLPerf inference loadgen using CM script automation
+
+on:
+  pull_request:
+    branches: [ "master", "dev" ]
```

Comment:
Do we have a dev branch? or do we plan to have one?

Reply:
Thank you @pgmpablo157321  for reviewing. I think we'll need a dev branch as we are going to use the master branch for submission. So, from the code freeze stage until submission we'll need to use a dev branch.